architecture/azure/messaging.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Azure messaging
+summary:
+reviewed: 2023-07-18
+callsToAction: ['solution-architect', 'poc-help']
+---
+
+Azure offers [several messaging services](https://learn.microsoft.com/en-us/azure/service-bus-messaging/compare-messaging-services), each built for specific purposes. The Particular Service Platform supports [Azure Service Bus and Azure Queue Storage](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-azure-and-service-bus-queues-compared-contrasted).
+
+### Azure Service Bus
+
+[Azure Service Bus](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview) is a fully-fledged enterprise messaging service with support for queuing, publish/subscribe, and more advanced integration patterns. It is designed to integrate applications or application components that may span multiple communication protocols, data contracts, trust domains, or network environments.
+
+Azure Service Bus has two pricing tiers: [Standard and Premium](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-premium-messaging). The Premium tier provides hardware-level isolation, which means each customer's workload runs on dedicated hardware. Premium tier uses _messaging units_ for scaling and pricing and guarantees predictable throughput and performance. The Standard tier offers pay-as-you-go pricing with quota-based throughput and non-guaranteed latency. Microsoft recommends running production systems on the Premium tier.
+
+:heavy_plus_sign: Pros:
+
+- Up to 100 MB message size on the Premium tier
+- Supports transactional message operations across multiple queues and topics (cross-entity transactions)
+- Scripted infrastructure deployment using the [NServiceBus ASB CLI](/transports/azure-service-bus/operational-scripting.md)
+- Supports the AMQP 1.0 protocol over TCP and WebSockets
+
+:heavy_minus_sign: Cons:
+
+- Some features are only available on the Premium tier
+- The maximum message size is 256 KB on the Standard tier
+- The maximum queue size is 80 GB on the Standard tier
```

Comment:
Azure and us say you shouldn't use Standard for production, so why should the limitations of Standard be a Con consideration?